diff -Naur opentelemetry-python-1.5.0-original/exporter/opentelemetry-exporter-jaeger/setup.cfg opentelemetry-python-1.5.0/exporter/opentelemetry-exporter-jaeger/setup.cfg
--- opentelemetry-python-1.5.0-original/exporter/opentelemetry-exporter-jaeger/setup.cfg	2021-08-26 05:24:39.000000000 -0400
+++ opentelemetry-python-1.5.0/exporter/opentelemetry-exporter-jaeger/setup.cfg	2022-03-11 14:59:27.813489101 -0500
@@ -36,11 +36,15 @@
 
 [options]
 python_requires = >=3.6
-
+package_dir=
+    =src
 packages=find_namespace:
 install_requires =
     opentelemetry-exporter-jaeger-proto-grpc == 1.5.0
     opentelemetry-exporter-jaeger-thrift == 1.5.0
 
+[options.packages.find]
+where = src
+
 [options.extras_require]
 test =
diff -Naur opentelemetry-python-1.5.0-original/exporter/opentelemetry-exporter-otlp/setup.cfg opentelemetry-python-1.5.0/exporter/opentelemetry-exporter-otlp/setup.cfg
--- opentelemetry-python-1.5.0-original/exporter/opentelemetry-exporter-otlp/setup.cfg	2021-08-26 05:24:39.000000000 -0400
+++ opentelemetry-python-1.5.0/exporter/opentelemetry-exporter-otlp/setup.cfg	2022-03-11 14:59:43.578684679 -0500
@@ -36,6 +36,11 @@
 
 [options]
 python_requires = >=3.6
+package_dir=
+    =src
 packages=find_namespace:
 install_requires =
     opentelemetry-exporter-otlp-proto-grpc == 1.5.0
+
+[options.packages.find]
+where = src
diff -Naur opentelemetry-python-1.5.0-original/exporter/opentelemetry-exporter-zipkin/setup.cfg opentelemetry-python-1.5.0/exporter/opentelemetry-exporter-zipkin/setup.cfg
--- opentelemetry-python-1.5.0-original/exporter/opentelemetry-exporter-zipkin/setup.cfg	2021-08-26 05:24:39.000000000 -0400
+++ opentelemetry-python-1.5.0/exporter/opentelemetry-exporter-zipkin/setup.cfg	2022-03-11 14:59:59.011876140 -0500
@@ -36,10 +36,15 @@
 
 [options]
 python_requires = >=3.6
+package_dir=
+    =src
 packages=find_namespace:
 install_requires =
     opentelemetry-exporter-zipkin-json == 1.5.0
     opentelemetry-exporter-zipkin-proto-http == 1.5.0
 
+[options.packages.find]
+where = src
+
 [options.extras_require]
 test =
