diff -Naur opentelemetry-python-1.6.1-original/exporter/opentelemetry-exporter-jaeger/setup.cfg opentelemetry-python-1.6.1/exporter/opentelemetry-exporter-jaeger/setup.cfg
--- opentelemetry-python-1.6.1-original/exporter/opentelemetry-exporter-jaeger/setup.cfg	2021-10-18 15:44:24.000000000 -0400
+++ opentelemetry-python-1.6.1/exporter/opentelemetry-exporter-jaeger/setup.cfg	2022-03-12 09:10:46.816517000 -0500
@@ -36,12 +36,16 @@
 
 [options]
 python_requires = >=3.6
-
+package_dir=
+    =src
 packages=find_namespace:
 install_requires =
     opentelemetry-exporter-jaeger-proto-grpc == 1.6.1
     opentelemetry-exporter-jaeger-thrift == 1.6.1
 
+[options.packages.find]
+where = src
+
 [options.extras_require]
 test =
 
diff -Naur opentelemetry-python-1.6.1-original/exporter/opentelemetry-exporter-otlp/setup.cfg opentelemetry-python-1.6.1/exporter/opentelemetry-exporter-otlp/setup.cfg
--- opentelemetry-python-1.6.1-original/exporter/opentelemetry-exporter-otlp/setup.cfg	2021-10-18 15:44:24.000000000 -0400
+++ opentelemetry-python-1.6.1/exporter/opentelemetry-exporter-otlp/setup.cfg	2022-03-12 09:10:49.827554250 -0500
@@ -36,11 +36,16 @@
 
 [options]
 python_requires = >=3.6
+package_dir=
+    =src
 packages=find_namespace:
 install_requires =
     opentelemetry-exporter-otlp-proto-grpc == 1.6.1
     opentelemetry-exporter-otlp-proto-http == 1.6.1
 
+[options.packages.find]
+where = src
+
 [options.entry_points]
 opentelemetry_traces_exporter =
     otlp = opentelemetry.exporter.otlp.proto.grpc.trace_exporter:OTLPSpanExporter
diff -Naur opentelemetry-python-1.6.1-original/exporter/opentelemetry-exporter-zipkin/setup.cfg opentelemetry-python-1.6.1/exporter/opentelemetry-exporter-zipkin/setup.cfg
--- opentelemetry-python-1.6.1-original/exporter/opentelemetry-exporter-zipkin/setup.cfg	2021-10-18 15:44:24.000000000 -0400
+++ opentelemetry-python-1.6.1/exporter/opentelemetry-exporter-zipkin/setup.cfg	2022-03-12 09:10:52.071582011 -0500
@@ -36,11 +36,16 @@
 
 [options]
 python_requires = >=3.6
+package_dir=
+    =src
 packages=find_namespace:
 install_requires =
     opentelemetry-exporter-zipkin-json == 1.6.1
     opentelemetry-exporter-zipkin-proto-http == 1.6.1
 
+[options.packages.find]
+where = src
+
 [options.extras_require]
 test =
 
